src/luamtev.c

```diff
@@ -202,6 +208,13 @@ int luaopen_hostcli(lua_State *L) {
     lua_setfield(L, -2, key);
   }
   lua_setglobal(L, "ENV");
+
+  lua_getglobal(L, "mtev");
+  mtevAssert(lua_istable(L, -1));
+  lua_getfield(L, -1, "printto");
+  mtevAssert(lua_isfunction(L, -1));
+  lua_pushstring(L, "stdout");
+  lua_call(L, 1, 0);
```

Comment:
So we change the default print target for luamtev apps here.

Is there a reason we don't globally change the print target for the whole mtev lua module?

Reply:
This globally changes it.

Reply:
right. But only for command line apps. The last comment explains the reasoning. OK.